components/Tabs/src/Theming/MDCTabBar+MaterialTheming.m

```diff
@@ -65,6 +66,8 @@ - (void)applySurfaceThemeWithColorScheme:(id<MDCColorScheming>)colorScheme {
       [colorScheme.onSurfaceColor colorWithAlphaComponent:kSurfaceThemeUnselectedOpacity];
   [self setTitleColor:unselectedTitleColor forState:MDCTabBarItemStateNormal];
   [self setImageTintColor:unselectedImageColor forState:MDCTabBarItemStateNormal];
+  self.bottomDividerColor =
```

Comment:
This probably shouldn't be enabled by default. The divider should only be enabled if a client needs it to separate the Tab Bar from other content.

Reply:
Got rid of this change 👍